dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -129,8 +135,8 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 
 				windowRects.forEach {
 					val rectMap = createMap()
-					rectMap.putInt("width", it.right - it.left)
-					rectMap.putInt("height", it.bottom - it.top)
+					rectMap.putInt("width", convertPixelsToDp(it.right - it.left))
+					rectMap.putInt("height",  convertPixelsToDp(it.bottom - it.top))
 					rectMap.putInt("x", it.left)
 					rectMap.putInt("y", it.top)
```

Comment:
x and y also need to be converted to DIPs (all coordinates are currently in pixels before your fix)

Reply:
oh my bad, doing this now